apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/constant/PermissionType.java

```diff
@@ -1,27 +1,32 @@
 package com.ctrip.framework.apollo.portal.constant;
 
+/**
+ * 权限类型
+ */
 public interface PermissionType {
 
+
   /**
-   * APP level permission
+   * 应用级别的权限
    */
-
   String CREATE_NAMESPACE = "CreateNamespace";
-
   String CREATE_CLUSTER = "CreateCluster";
 
   /**
    * 分配用户权限的权限
    */
   String ASSIGN_ROLE = "AssignRole";
 
+
   /**
-   * namespace level permission
+   * namespace级别的权限
```

Comment:
I think it's ok to leave the comment as English, as it's our goal to make apollo readable to world wide developers